src/components/react-json-schemaForm/TableField/TableField.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Thead from './Thead';
+import Tbody from './Tbody';
+
+import './styles.scss';
+
+const TableField = props => {
+  const {
+    schema: { title },
+  } = props;
+
+  return (
+    <fieldset className="field field-array field-array-of-object">
+      <legend>{title}</legend>
+      <div className="details__table tableField">
+        <table>
+          <Thead {...props} />
+          <Tbody {...props} />
+        </table>
+      </div>
+    </fieldset>
+  );
+};
+
+TableField.propTypes = {
+  onChange: PropTypes.func,
+  schema: PropTypes.shape({
+    items: PropTypes.shape({
+      properties: PropTypes.shape({}),
+      required: PropTypes.arrayOf(PropTypes.string),
+    }),
+    title: PropTypes.string,
+  }),
+};
+
+TableField.defaultProps = {
+  schema: {
+    items: {
+      properties: {},
+      required: [],
+    },
+    title: '',
+  },
+  onChange: () => {},
+};
```

Comment:
Shouldn't those props be imported from `<Thead />` or `<Tbody />` component instead of being duplicated here?

Reply:
Understood,
How do you suggest the propType/defaultProp of `title` ? Something like ⬇️ 
```
const allDefaultProps = { ...Tbody.defaultProps, ...Thead.defaultProps }
TableField.defaultProps = {
  ...allDefaultProps,
  schema: {
    ...allDefaultProps.schema,
    title: ''
  }
};
```
Maybe, it lose some readability

Reply:
I think I would personaly not bother with propTypes, and only provide default value directly when destructuring:
```jsx
const TableField = props => {
  const {
    schema: { title = '' },
  } = props;

  // ...
```

*(I also think that `schema` and `uiSchema` propTypes shouldn't be defined each type a Component use them. They should be defined only once, then import and used when needed. Also, tools for generating propTypes from JSONSchema exist)*

Reply:
Ok done,
Finally, i didn't set a default prop to title in benefit of a conditional display of this title